themes/edge.edx.org/lms/static/sass/partials/base/_certificates.scss

```diff
@@ -0,0 +1,27 @@
+// Certificate overrides for edge.edx.org
+
+.certificate-card {
+    &.mode-honor, &.mode-audit {
+        border-color: $honor-mode-color;
+
+        .card-logo {
+            background-image: url('#{$static-path}/images/certificates/honor.png');
+        }
+    }
+
+    &.mode-verified {
+        border-color: $verified-mode-color;
+
+        .card-logo {
+            background-image: url('#{$static-path}/images/certificates/verified.png');
+        }
+    }
+
+    &.mode-professional {
+        border-color: $professional-certificate-color;
```

Comment:
This color name seems like it might apply to `professionalcert-program` and not `professional-education`, but is not clear.  It would be great to have the language around this clarified.

Reply:
This should all be clearer (or refactored if not) when we support program certificates. For the moment, this is the color for professional course certificates. I think that the file being renamed to certificates/professional.png should help with the naming confusion.

Reply:
Not that two of the pngs seemed to have nothing but a color change. That's not an accessible difference unfortunately, and I barely noticed the difference. As you mentioned, that's an upcoming problem and not a current problem.
Also, it would be cool if someone could see a key or something somewhere that tells what the images mean. I think that is onle there for screen readers?

Reply:
@andy-armstrong: What's the ticket for handling Program Certificates?  I think we need to add the following comments to that ticket:
1. Color difference alone is not enough to differentiate professional from professional-program.
2. Do we need some way for users to know what each icon means?  (Separate PR or this one?)
3. When we were together we found code related to translating professional cert modes on the Program Dashboard I think?  Do you need a note around moving/fixing this code?  Is that part of the future PR, or this one?

Reply:
@robrap Good point. I've added this to the ticket:

https://openedx.atlassian.net/browse/LEARNER-2380

Reply:
Regarding the two different certificate types, they are in different sections so maybe it isn't terrible if the icons are indistinguishable. I agree it would be better if there was a visual way to distinguish them though.

Reply:
I see.  If they are in two different sections, I think that would be enough to distinguish them. I added a note to the ticket.

@andy-armstrong: I don't see an acknowledgement of point 3.  Let me know if it was unclear and maybe I can help you remember what I am vaguely trying to describe.  Thanks.

Reply:
I was thinking that point 3 was unrelated so I was going to create a different ticket. However, I now think it is close enough that it can be part of the clean up of the API.

Reply:
@andy-armstrong: I'm good with whatever you decide.  I just need to hear your thoughts to know it wasn't missed/lost.  Thanks!

Reply:
I added it to the ticket.